src/assets/src/components/queueManager.tsx

```diff
@@ -212,7 +212,13 @@ function QueueManager(props: QueueManagerProps) {
             </Table>
         )
         : (
-            <Alert variant="dark">No meetings in queue.</Alert>
+            <>
+            <hr/>
```

Comment:
Nit-pick: you could indent a bit more here for readability.

Reply:
Huh, thanks for explaining that. Your approach seems fine. I commented because in general I find it easier to read markup when the levels are clear. I've often used `<div>` instead of `<>` because I thought (I guess erroneously) that they were just converted into `div`s. I guess there's no harm in overusing `div`s, but I'm glad you pointed out these nuances.